404.html

```diff
@@ -1,3 +1,7 @@
 NOVTEST
 1
 Minor tweak
+2
+Now that I have push access
```

Comment:
```suggestion
Now that I have push access
```
What has been given can be taken away